packages/app/src/app/vscode/index.ts

```diff
@@ -316,6 +316,21 @@ class VSCodeManager {
     });
   }
 
+  /*
+    Allows to change code of the editor. This happens when there is inconsistency in LIVE mode
+    and server requires a full code update
+  */
+  changeCode(moduleShortid: string, code: string) {
+    try {
+      (window as any).CSEditor.editor
+        .getActiveCodeEditor()
+        .getModel(moduleShortid)
```

Comment:
Whoah this really works?! I'm not sure if this is possible 🤔

Reply:
Yeah, for sure, just run my TEST function 😄